csfieldguide/templates/interactives/picture-memory.html

```diff
@@ -0,0 +1,247 @@
+          {% svg 'picture-memory/icons8-scissors' %}
+        </div>
+        <div id="item-duck" class="stm-item col-3 d-none">
+          {% svg 'picture-memory/icons8-duck' %}
+        </div>
+        <div id="item-guitar" class="stm-item col-3 d-none">
+          {% svg 'picture-memory/icons8-guitar' %}
+        </div>
+
+        <div id="item-bike" class="stm-item col-3 d-none">
+          {% svg 'picture-memory/icons8-bicycle' %}
+        </div>
+        <div id="item-camera" class="stm-item col-3 d-none">
+          {% svg 'picture-memory/icons8-camera' %}
+        </div>
+        <div id="item-football" class="stm-item col-3 d-none">
+          {% svg 'picture-memory/icons8-soccer-ball' %}
+        </div>
+
+        <div id="item-book" class="stm-item col-3 d-none">
+          {% svg 'picture-memory/icons8-book' %}
+        </div>
+        <div id="item-toothbrush" class="stm-item col-3 d-none">
+          {% svg 'picture-memory/icons8-electric-toothbrush' %}
+        </div>
+        <div id="item-teapot" class="stm-item col-3 d-none">
+          {% svg 'picture-memory/icons8-teapot' %}
+        </div>
+        <div id="item-calculator" class="stm-item col-3 d-none">
+          {% svg 'picture-memory/icons8-calculator' %}
+        </div>
+
+        <div id="item-forklift" class="stm-item col-3 d-none">
+          {% svg 'picture-memory/icons8-fork-lift' %}
+        </div>
+        <div id="item-sock" class="stm-item col-3 d-none">
+          {% svg 'picture-memory/icons8-socks' %}
+        </div>
+        <div id="item-cactus" class="stm-item col-3 d-none">
+          {% svg 'picture-memory/icons8-cactus' %}
+        </div>
+
+        <div id="item-cup" class="stm-item col-3 d-none">
+          {% svg 'picture-memory/icons8-cup' %}
+        </div>
+        <div id="item-envelope" class="stm-item col-3 d-none">
+          {% svg 'picture-memory/icons8-envelope' %}
+        </div>
+        <div id="item-glasses" class="stm-item col-3 d-none">
+          {% svg 'picture-memory/icons8-glasses' %}
+        </div>
+        <div id="item-backpack" class="stm-item col-3 d-none">
+          {% svg 'picture-memory/icons8-school-backpack' %}
+        </div>
+
+        <div id="item-t-shirt" class="stm-item col-3 d-none">
+          {% svg 'picture-memory/icons8-t-shirt' %}
+        </div>
+        <div id="item-chair" class="stm-item col-3 d-none">
+          {% svg 'picture-memory/icons8-chair' %}
+        </div>
+        <div id="item-sheep" class="stm-item col-3 d-none">
+          {% svg 'picture-memory/icons8-sheep' %}
+        </div>
+      </div>
+
+      <div id="incorrect-items-div" class="text-center d-none">
+        <p id="incorrect-items-text" class="text-center">{% trans "You thought you saw the following items when they were never shown!" %}</p>
+        <p id="incorrect-items" class="text-center"></p>
+      </div>
+
+      <div id="answer-input" class="text-center col-12 d-none">
```

Comment:
Consider adding some sort of instructions to this phase, like 'What do you remember?' or something. Also consider arranging them vertically in alphabetical order rather than horizontally, as it looks more like 3 distinct columns than 7 rows